sapp/ui/frontend/src/Filter.js

```diff
@@ -956,6 +957,13 @@ const Filter = (props: {refetch: any, refetching: boolean}) => {
     window.sessionStorage.setItem('filter', JSON.stringify(currentFilter));
   }, [currentFilter]);
 
+  const clearFilter = () => {
```

Comment:
Curious why we have two `clearFilter` functions (the one here and one introduced in [PR#29](https://github.com/facebook/sapp/pull/29/files#diff-356ada66e790543c2d35ebb56b522911990d711a0f46478a7fc45f4ddb89a49fR923)). Are we unable to use `window.sessionStorage.remove('filter')` here, because it would require calling `window.location.reload()` for the UI to update with the changes?

Would there be a way to only have one `clearFilter` function we use for this PR and PR#29?

Reply:
One of the function deletes the filter from local session storage and the other removes all currently applied filters.

Reply:
React noob here, but would something like this work instead?
```javascript
 const clearFilter = (refreshUI: boolean = true) => {
    window.sessionStorage.removeItem('filter');
    if (refreshUI) {
      props.refetch();
      setCurrentFilter(emptyFilter);
    }
  };
```

For PR#29, we can call the same function, but pass in `refreshUI=false` instead

Reply:
I will try